.github/workflows/build-workshop-site.yml

```diff
@@ -11,7 +11,8 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Install Hugo dependency
-        run: sudo apt-get install hugo
+        working-directory: ./cdkworkshop.com
+        run: sudo tar -C /usr/local/bin -xzf hugo/hugo_*_Linux-64bit.tar.gz hugo
```

Comment:
For the future, can we store this on s3 or something? It seems silly that we're checking a source tarball into the repo and not doing something like using git submodules.

Reply:
It's a binary, but yeah. 

There's two or three basic options:

* We pull it from GitHub via the release, updating the release on occasion, and adding some mechanism to verify the download. 
* We build Hugo locally (bad idea, takes a lot of time)
* We use a [GH action](https://github.com/peaceiris/actions-hugo) to pull in the latest version of Hugo (or a tagged version) instead. 

I think the last is the best option if we can/are allowed to use a well-known community trusted action.

Reply:
I'll ask for forgiveness rather than permission here.